components/org.wso2.carbon.identity.oidc.dcr/src/main/java/org/wso2/carbon/identity/oidc/dcr/processor/OIDCDCRProcessor.java

```diff
@@ -54,9 +54,14 @@
         boolean isIdentityConnectDCREnabled = Boolean.parseBoolean(IdentityUtil.getProperty(ENABLE_OIDC_DCR_CONFIG));
         if (!isIdentityConnectDCREnabled) {
             if (log.isDebugEnabled()) {
-                log.debug("Identity Connect DCR Endpoint is disabled.");
+                log.debug("Identity Connect DCR endpoint was deprecated. To enable the DCR API endpoint " +
```

Comment:
change as "Identity Connect DCR endpoint is deprecated and hence disabled by default in the server. To enable it add the following config..."

Reply:
Fixed https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1407/commits/515007e26b0cd144ab864d90dfd2e76cfc8f9c2c